scripts/download.pl

```diff
@@ -100,6 +123,7 @@ sub download
 {
 	my $mirror = shift;
 	my $download_filename = shift;
+	my @mrs = @_;
```

Comment:
lets make this more readable.
```suggestion
	my @additional_mirrors= @_;
```